snapcraft/cli/_options.py

```diff
@@ -52,7 +52,13 @@ def _add_build_options(func):
     return _add_build_options
 
 
-def get_project_options(**kwargs):
+def get_project(**kwargs):
+    skip_snapcraft_yaml = kwargs.pop('skip_snapcraft_yaml', False)
```

Comment:
It would be cleaner and more intuitive to include this as an actual kwarg so you don't need to pop it:

```python
def get_project(*, skip_snapcraft_yaml: bool=False, **kwargs):
    # ...
```